client/reader/start/style.scss

```diff
@@ -81,14 +81,26 @@
 		font-family: $serif;
 		font-size: 12px;
 		margin: 10px 0 -3px;
-		max-height: none;
-	}
+		max-height: 65px;
 
-	.post-excerpt.is-long {
+		@include breakpoint( ">480px" ) {
+			max-height: 56px;
+		}
 
 		&:after {
-			display: none;
+			@include long-content-fade;
+			height: 25px;
+			position: absolute;
+				bottom: 0;
+				top: 40px;
+			width: 65%;
+
+			@include breakpoint( ">680px" ) {
```

Comment:
My bad, I mistyped it. Fixed in: https://github.com/Automattic/wp-calypso/pull/6186/commits/b553a59ed7d4aea8fa2ca78748ff920caa3d49ca